### PR TITLE
Orderbook guide 1

### PR DESCRIPTION
**Pull request by Lakshya18jul**

### CODE REVIEW

File: src/orderbook-basics.yaml [hunk 1]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
+      High levels of transparency offered by the CLOB model allow customers to monitor market depth in real time. Although CLOB has very powerful features, integrating it into an on-chain system is exceedingly difficult. The key reasons for this difficulty are order books' high throughput and cheap execution costs. Therefore, low gas costs and rapid transactions are necessary for a CLOB to operate on-chain.
+
+      Many of the first DEX protocols were conscious of these challenges and made an effort to replace the traditional CLOB paradigm with an AMM. By employing their AMM technology, Uniswap's Defi users were able to complete transactions rapidly and affordably.
     stepItems: []
-  - uuid: 7994939b-5188-4951-aaf1-e3a268725fb6
+  - uuid: a0271380-9bc8-49c0-b825-f6e9149bb4fe
     name: Orderbook vs AMM
     content: |
-      All the use cases
+      Orderbook and Automated market maker are two ost popular trading models. They primarly differentiate on the basis on structure and underlying working principle.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      Orderbook and Automated market maker are two most popular trading models. They primarily differentiate on the basis on structure and underlying working principle.
```

File: src/orderbook-basics.yaml [hunk 2]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
+
+      ### Structural Differences 
+      
+      An order book works on the principle of FIFO- First In First Out. An algorithm, in the Order Book, organises the list of traders who submitted their intent to transact to buy or to sell and subsequently organises them in a database i.e. order book. This database is then made public to all the participating traders.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      An order book works on the principle of Price and Time. An algorithm, in the Order Book, organizes the list of traders who submitted their intent to transact to buy or to sell and subsequently organizes them in a database i.e. order book. This database is then made public to all the participating traders.
```

File: src/orderbook-basics.yaml [hunk 3]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
+
+      AMM is a trading model that is automated.The main distinction between the two is that whereas AMM employs two-sided liquidity pools in an order book where both buyers and sellers serve as the liquidity custodian. Given its support for intermediary intervention and manual components, the Order Book trading model is popular among centralised exchanges, whereas AMM is the fundamental protocol utilised by decentralised exchanges to do away with middlemen when trading crypto assets.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      AMM is a trading model that is automated. The main distinction between the two is that AMM employs two-sided liquidity pools, whereas in an order book both buyers and sellers serve as the liquidity custodian. Given its support for intermediary intervention and manual components, the Order Book trading model is popular among centralized exchanges, whereas AMM is the fundamental protocol utilized by decentralized exchanges to do away with middlemen when trading crypto assets.
```

File: src/orderbook-basics.yaml [hunk 4]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
+
+      ### Working Principle Differences
+
+      The orderbook principle comes into play when  the exchange encounters overlapping orders in the order book especially when there are multiple requests regarding the same value and quantity of a traded asset, in such cases the exchange that uses an order book concept steps into matching these orders manually using the FIFO.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      The orderbook principle comes into play when  the exchange encounters overlapping orders in the order book especially when there are multiple requests regarding the same value and quantity of a traded asset, in such cases the exchange that uses an order book concept steps into matching these orders first by price and then for the same price match, it gives priority to the orders that came in first/before. 
```

File: src/orderbook-basics.yaml [hunk 5]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
     stepItems: []
-  - uuid: f317cc4e-1f45-496d-b01a-61ca28156997
-    name: AMM on top of Order Book
+  - uuid: 6fc6f39c-81a8-4796-aade-bfff10267e15
+    name: Advantages of CLOB
     content: |
-      AMM on top of Order Book
+      Some of advantages of CLOB model are:
+
+      * Users can execute orders using smart contracts in Serum's fully decentralised, on-chain order book while keeping pricing and order size flexibility.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      * Users can execute orders using smart contracts in Serum's fully decentralized, on-chain order book while keeping pricing and order size flexibility.
```

File: src/orderbook-basics.yaml [hunk 6]
```diff
@@ -12,23 +12,90 @@ publishStatus: Live
...
+
+      * Serum enhances composability as it can be used by other protocols in the area to bootstrap liquidity and provide matching services.
+
+      * The fast throughput and low transaction costs of the Solana network assist the Serum order book in minimising capital inefficiencies and liquidity segmentation.
+
+      * Serum's CLOB is built to be asset agnostic and has additional strong features like cross-chain swaps. It may create an order book that corresponds to any trading product offered by Solana, including options and futures (e.g. ETH and BTC).
+
+      * Due to Serum's flexible protocol, a wide range of participants and applications can share middleware in one place. This is made possible by the backend matching engine's versatility, which allows it to be extended to almost any financial or non-financial item.
+      
+      * Other applications can connect to Serum and leverage its infrastructure to build their own projects thanks to on-chain CLOB, such as a trading application that utilises Serum's liquidity.
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      * Other applications can connect to Serum and leverage its infrastructure to build their own projects thanks to on-chain CLOB, such as a trading application that utilizes Serum's liquidity.
```

File: src/orderbook-basics.yaml (hunk 1 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 2 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 3 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 4 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 5 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 6 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml [hunk 7]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
-    name: What is AMM
+
+  - uuid: 0d22ec66-17d5-470e-81d9-43b207e344f4
+    name: Evaluation
     content: |
-      https://ga245xphr2gqozusue66acdwqe2jdoflnonum32n5rvyr2dy.arweave.net/MDXO3eeOjQdmkq_E94Ah2gTS_RuKtrm0ZvTexriOh4M
+        
+    stepItems:
+      - uuid: 12a809d7-34b0-4edd-a4d1-d62a0e1354ec
+        content: What is the difference between IFB and RFQ?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: Use full forms of IFB and RFQ.

File: src/orderbook-basics.yaml [hunk 8]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+          - content: Uniswap
+            key: A
+          - content: Curve
+            key: B
+          - content: Serum
+            key: C
+          - content: Balancer
+            key: D
+      - uuid: 984d2c1a-b521-433f-98ca-3a84035942bb
+        content: Which of the following DEXs use CLOB?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: Use full form of DEX and CLOB

File: src/orderbook-basics.yaml [hunk 9]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
     stepItems: []
-  - uuid: 7994939b-5188-4951-aaf1-e3a268725fb6
+
+  - uuid: 4dfc846e-09b4-4c51-8374-dfd1e3e5048e
+    name: Evaluation
+    content: |
+
+    stepItems:
+      - uuid: c7b5cdea-2928-4dd7-8da2-d786fd98187f
+        content: Serum uses which of the following builders?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
        content: Serum uses which of the following blockchains?
```

File: src/orderbook-basics.yaml [hunk 10]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+          - content: Ethereum
+            key: A
+          - content: Hyperledger
+            key: B
+          - content: Solana
+            key: C
+          - content: R3
+            key: D
+      - uuid: 44a8a64e-2fb5-43c9-9392-36a61334831d
+        content: Serum CLOB works on the basis of-
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: CLOB uses both price and time so its not purely FIFO

Comment (RobinNagpal) [reply src/orderbook-basics.yaml]: Also use "CLOB" -> "Center Limit Order Book"

File: src/orderbook-basics.yaml [hunk 11]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+      This ought to provide a crystal-clear and upbeat image of Serum DEX's potential as the matching engine powering Solana-based projects that make use of the DEX's matching service, liquidity, and price data.   
+    stepItems: []
+  
+  - uuid: c0207687-5da1-4450-b395-c1efe7ce0092
+    name: Evaluation
+    content: |
+
+    stepItems:
+      - uuid: e00fa4ba-a439-4b38-bfb4-1545770a9343
+        content: Which property of Solana helps building over the Serum?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
        content: Which property of Project Serum helps other projects build on top of Serum?
```

File: src/orderbook-basics.yaml [hunk 12]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+          - content: Fast Throughput 
+            key: A
+          - content: Low Transaction cost 
+            key: B
+          - content: Large Network
+            key: C
+          - content: High Composibility
+            key: D
+      - uuid: d93f538b-0acd-4401-b817-94728867d81b
+        content: Which of the following Serum DEXs is building better lending and borrowing?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
        content: Which of the following protocol is building lending and borrowing and is build on top of Serum?
```

File: src/orderbook-basics.yaml [hunk 13]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+      
+      - uuid: 5f5b0575-7d48-4365-a523-2033a64fabb4
+        content: What do you mean by cross-chain swaps?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: Swapping within a blockchain ecosystem
+            key: A
+          - content: Swapping across different Cryptos using smart contracts
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: What do we mean by `different Cryptos` here?

File: src/orderbook-basics.yaml [hunk 14]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+            key: A
+          - content: Quantity
+            key: B
+          - content: Direction
+            key: C
+          - content: Transaction Time
+            key: D
+
+      - uuid: 02db90e6-1301-4bdf-b04e-b97eaafa22e1
+        content: How does Solana empower Serum CLOB?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: use full form for CLOB

File: src/orderbook-basics.yaml [hunk 15]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+    stepItems:
+      - uuid: 53c58fcf-8101-48f8-9681-899a4b173a0b
+        content: CLOB enhances the liquidity by specifying ____ of transaction.
+        answerKeys:
+          - A
+          - B
+          - C
+        type: MultipleChoice
+        choices:
+          - content: Price
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
          - content: Limit Price 
```

File: src/orderbook-basics.yaml [hunk 16]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
-    name: AMM on top of Order Book
+
+  - uuid: 3cf47c18-7544-4491-ad86-e5220a0c8a73
+    name: Evaluation
     content: |
-      AMM on top of Order Book
+
+    stepItems:
+      - uuid: 9a837714-e236-4ee5-b084-2d0fe5729b03
+        content: How does CLOB and AMM differ on the basis of contributors?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]: Use full forms for CLOB and AMM

File: src/orderbook-basics.yaml (hunk 7 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 8 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 9 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 10 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Made it more intuitive

File: src/orderbook-basics.yaml (hunk 11 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 12 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 13 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Different tokens

File: src/orderbook-basics.yaml (hunk 14 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 15 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml (hunk 16 above)
Comment (Lakshya18jul) [reply src/orderbook-basics.yaml]: Done

File: src/orderbook-basics.yaml [hunk 17]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
-    name: What is AMM
+
+  - uuid: 0d22ec66-17d5-470e-81d9-43b207e344f4
+    name: Evaluation
     content: |
-      https://ga245xphr2gqozusue66acdwqe2jdoflnonum32n5rvyr2dy.arweave.net/MDXO3eeOjQdmkq_E94Ah2gTS_RuKtrm0ZvTexriOh4M
+        
+    stepItems:
+      - uuid: 12a809d7-34b0-4edd-a4d1-d62a0e1354ec
+        content: What is the difference between Invitation for Bid and Request For Quote?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
        content: What is the difference between Invitation for Bid and Request For Quote(RFQ)?
```

File: src/orderbook-basics.yaml [hunk 18]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
-    name: What is AMM
+
+  - uuid: 0d22ec66-17d5-470e-81d9-43b207e344f4
+    name: Evaluation
     content: |
-      https://ga245xphr2gqozusue66acdwqe2jdoflnonum32n5rvyr2dy.arweave.net/MDXO3eeOjQdmkq_E94Ah2gTS_RuKtrm0ZvTexriOh4M
+        
+    stepItems:
+      - uuid: 12a809d7-34b0-4edd-a4d1-d62a0e1354ec
+        content: What is the difference between Invitation for Bid and Request For Quote(RFQ)?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
        content: What is the difference between Invitation for Bid(IFB) and Request For Quote(RFQ)?
```

File: src/orderbook-basics.yaml [hunk 19]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+          - content: Price Discovery
+            key: A
+          - content: Market Liquidity
+            key: B
+          - content: Market Fragmentation
+            key: C
+          - content: None of these
+            key: D
+      - uuid: 6f11766f-8bfb-4a62-ab2b-f4e91ca299fa
+        content: Which of the following DEXs use AMM?
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml:72]: Use full forms for DEX and AMM

File: src/orderbook-basics.yaml [hunk 20]
```diff
@@ -12,23 +12,371 @@ publishStatus: Live
...
+            key: B
+          - content: Constant Market Maker Model
+            key: C
+          - content: Last In First Out
+            key: D
+
+  - uuid: 6fc6f39c-81a8-4796-aade-bfff10267e15
+    name: Advantages of CLOB
+    content: |
+      Some of advantages of CLOB model are:
```
Comment (RobinNagpal) [inline src/orderbook-basics.yaml]:
```suggestion
      Some of advantages of Center Limit Order Book(CLOB) model are:
```

**Pull request merged**